src/components/ReportingConfig/SFTPDeliveryMethodForm.jsx

```diff
@@ -1,111 +1,94 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Input, ValidationFormGroup } from '@edx/paragon';
+import { Form } from '@edx/paragon';
+import { Error } from '@edx/paragon/icons';
 
 const SFTPDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
   const [checked, setChecked] = useState(false);
 
+  const renderField = data => (
+    <Form.Group
+      key={data.key}
+      controlId={data.key}
+      isInvalid={invalidFields[data.key]}
+    >
+      <Form.Label htmlFor={data.key}>{data.label}</Form.Label>
```

Comment:
`htmlFor` is unnecessary since its applied programmatically via `Form.Group`'s `controlId`